src/helpers/console.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-console */
+/* eslint-disable no-restricted-globals */
+
+/**
+ * "In Internet Explorer 9 (and 8), the console object is only exposed when the developer tools are opened
+ * for a particular tab."
+ *
+ * Source: https://stackoverflow.com/a/5473193
+ */
+
+import {isDefined} from './mixed';
+
+/**
+ * Logs message to the console if the `console` object is exposed.
+ *
+ * @param {...String} messages Messages which will be logged.
```

Comment:
Maybe `args` instead of `messages`?
Arguments in `log/warn/info` could be `any`.